src/components/base/GrowScroll/index.js

```diff
@@ -47,7 +50,7 @@ class GrowScroll extends PureComponent<Props> {
 
     const scrollContainerStyles = {
       overflowY: 'scroll',
-      marginRight: `-80px`,
+      marginRight: `-${80 + scrollbarWidth}px`,
```

Comment:
use directly `marginRight: -80 + scrollbarWidth`

Reply:
not the same thing tho it's more like `-80-scrollbarWidth` , i think it's less readable tho,